backend/canisters/group/impl/src/updates/c2c_join_group.rs

```diff
@@ -93,7 +94,12 @@ fn c2c_join_group_impl(args: Args, runtime_state: &mut RuntimeState) -> Response
         mute_notifications: runtime_state.data.is_public,
```

Comment:
The `min_visible_event/message_index` should come from the invitation

Reply:
Good catch